workitem.go

```diff
@@ -320,6 +321,10 @@ func ConvertJSONAPIToWorkItem(appl application.Application, source app.WorkItem2
 			target.Type = source.Relationships.BaseType.Data.ID
 		}
 	}
+
+	if title, ok := source.Attributes[workitem.SystemTitle]; !ok || title == "" {
```

Comment:
ha, you mean we need to use the WIT to see if the `system.title` attribute is required ?